style.css

```diff
@@ -0,0 +1,81 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+body {
+  background-color: #e5e5e5;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin: 15%;
+}
+
+.home-page {
+  background-color: #fff;
+  border-radius: 0 0 0 5%;
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+}
+
+.log {
+  height: 30px;
+  width: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.hamburger {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 12px 0 10px;
+  padding: 0 48px 0 48px;
+}
```

Comment:
- [ ] Well done 👏🏻 you made the changes as per the previous code reviewer's request, but it seems like you didn't make all the changes for your **header** because your previous code reviewer mentioned regarding your design is not look good and aligned properly. Fortunately, I have found the issue and below is the suggestion for solving the issue.

```suggestion
.hamburger {
  display: flex;
  justify-content: space-between;
  padding: 1rem;
}
```
###  Design Comparison

<table>
<tr>
<th>Before Suggestion</th>
<th>After Suggestion</th>
</tr>

<tr>
<td>

![image](https://github.com/medijdir/portfolio_project/assets/66476329/6bc890ed-fe9b-440a-bf10-aef70a5c6396)


</td>

<td>

![Screenshot from 2023-06-07 13-54-15](https://github.com/medijdir/portfolio_project/assets/66476329/f1578279-2c6c-495a-9244-9d025a0f9533)


</td>
</tr>
</table>